### ISSUE TITLE
Add SSH Support

### ISSUE THREAD

**Issue by Ajitpals** [dadoonet/fscrawler#12]
You can now index files remotely using SSH:
- FS URL: `/tmp3`
- Server: `mynode.mydomain.com`
- Username: `username`
- Password: `password`
- Protocol: `ssh` (default to `local`)
- Update Rate: every hour (60 \* 60 \* 1000 = 3600000 ms)
- Get only docs like `*.doc`, `*.xls` and `*.pdf`

``` sh
curl -XPUT 'localhost:9200/_river/mysshriver/_meta' -d '{
  "type": "fs",
  "fs": {
    "url": "/tmp3",
    "server": "mynode.mydomain.com",
    "username": "username",
    "password": "password",
    "protocol": "ssh",
    "update_rate": 3600000,
    "includes": [ "*.doc" , "*.xls", "*.pdf" ]
  }
}'
```

**Comment by AncKal94** [comment]
Do we have integration of microsoft sharepoint with fscrawler as well?
fs:
  url: "https://infosystechnologies.sharepoint.com/sites/RCP_testing/Shared%20Documents/General/Testing"
just providing the sharepoint link here is giving error like
13:51:15,152 WARN  [f.p.e.c.f.FsParserAbstract] Error while crawling https://infosystechnologies.sharepoint.com/sites/RCP_testing/Shared%20Documents/General/Testing: https://infosystechnologies.sharepoint.com/sites/RCP_testing/Shared%20Documents/General/Testing doesn't exists.

**Comment by dadoonet** [comment]
No. It would need another source to do that properly. 
But https://www.elastic.co/workplace-search supports it I think.
As my intention is to connect FSCrawler to Workplace Search, I think it makes sense to look at this.

**Comment by AncKal94** [comment]
@dadoonet I tried implementing elastic workplace search in linux RHEL server referring the link:
https://www.elastic.co/downloads/enterprise-search

The Red Hat linux server is a restricted one and we can't directly download anything in it.
So, we downloaded linux elastic search 7.6.0.tar.gz and enterprise search 7.6.0 in our local windows system.
Unzipped those tar files in the linux server.
But not understanding how to start elastic search in linux server.
I have attached the output that i got after running ./elasticsearch.
Also how can i change the host address
[elastic_linux.txt](https://github.com/dadoonet/fscrawler/files/4492923/elastic_linux.txt)
